code/src/components/TaskList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import styled from 'styled-components/macro'
+
+import tasks from '../reducers/tasks'
+import EmptyStateAnimation from './EmptyStateAnimation'
+
+const TaskContainer = styled.div`
+    overflow: auto;
+    width: 100%;
+`
```

Comment:
Maybe add a max-width here, the textinput field get a bit large on the laptop 💻 (and I know I messed your styling in the last minute yesterday haha, sooorrryyy!) 🙇‍♀️